app/src/main/java/com/axiel7/moelist/ui/season/composables/SeasonChartFilterSheet.kt

```diff
@@ -49,6 +51,14 @@ fun SeasonChartFilterSheet(
     sheetState: SheetState,
     bottomPadding: Dp = 0.dp
 ) {
+
+    val scrollState = rememberLazyListState()
+
+    LaunchedEffect(key1 = true) {
+        scrollState.scrollToItem(SeasonChartUiState.years.indexOf(uiState.season.year))
```

Comment:
before calling `scrollToItem`, check if the found index is -1 so it doesn't crash